TwipsterApp/Models/PasswordValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace TwipsterApp.Models
+{
+    public class PasswordValidator : IValidator
+    {
+        public bool Validate(User user, string lineToValidate)
```

Comment:
You should define whether function returns bool as validation or throw an exception in case of error occuring. (Only one)